pages/02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -785,8 +1035,8 @@ div.banner[-ext-matches-css-before="content: block me"]
 In most cases, the basis and cosmetic rules are enough to filter ads. But sometimes it is necessary to change the HTML-code of the page itself before it is loaded. This is when you need filtering rules for HTML content. They allow to indicate the HTML elements to be cut out before the browser loads the page.
 
 > #### Compatibility with different versions of AdGuard
-> Rules are supported by AdGuard for Windows, Mac and Android (you have to set "filtering method" for "High-quality" in Android).
-> This type of rules don't work in browsers extension, because it is unable to modify content on network level.
+> Rules are supported by AdGuard for Windows, Mac, Android and by the AdGuard's Firefox add-on.
+> This type of rules don't work in other browsers extension, because it is unable to modify content on network level.
```

Comment:
This type of rules don't work in extensions for other browsers because they are unable to modify content on network level.

probably